ARM Builds Failing

1>  pch.cpp
1>c:\program files (x86)\windows kits\10\include\10.0.10586.0\um\dbgprop.h(1387): error C2061: syntax error: identifier 'DebugPropertyInfo'
1>c:\program files (x86)\windows kits\10\include\10.0.10586.0\um\dbgprop.h(1405): error C2061: syntax error: identifier 'DebugPropertyInfo'
1>c:\program files (x86)\windows kits\10\include\10.0.10586.0\um\dbgprop.h(1412): error C2061: syntax error: identifier 'DebugPropertyInfo'
1>C:\Program Files (x86)\Windows Kits\10\Include\10.0.10586.0\um\activdbg.h(9409): error C2061: syntax error: identifier 'DebugStackFrameDescriptor'
1>C:\Program Files (x86)\Windows Kits\10\Include\10.0.10586.0\um\activdbg.h(9416): error C2061: syntax error: identifier 'DebugStackFrameDescriptor'
1>C:\Program Files (x86)\Windows Kits\10\Include\10.0.10586.0\um\activdbg.h(9422): error C2061: syntax error: identifier 'DebugStackFrameDescriptor64'
1>C:\Program Files (x86)\Windows Kits\10\Include\10.0.10586.0\um\activdbg.h(9429): error C2061: syntax error: identifier 'DebugStackFrameDescriptor64'

--- a/ReactWindows/ChakraBridge/ChakraBridge.vcxproj
+++ b/ReactWindows/ChakraBridge/ChakraBridge.vcxproj
@@ -163,7 +163,7 @@
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|ARM'">
     <ClCompile>
       <PrecompiledHeader>Use</PrecompiledHeader>
-      <PreprocessorDefinitions>_WINRT_DLL;NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_WINRT_DLL;USE_EDGEMODE_JSRT;NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <PrecompiledHeaderOutputFile>$(IntDir)pch.pch</PrecompiledHeaderOutputFile>
       <AdditionalUsingDirectories>$(WindowsSDK_WindowsMetadata);$(AdditionalUsingDirectories)</AdditionalUsingDirectories>
@@ -193,7 +193,7 @@
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
     <ClCompile>
       <PrecompiledHeader>Use</PrecompiledHeader>
-      <PreprocessorDefinitions>_WINRT_DLL;NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_WINRT_DLL;USE_EDGEMODE_JSRT;NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <PrecompiledHeaderOutputFile>$(IntDir)pch.pch</PrecompiledHeaderOutputFile>
       <AdditionalUsingDirectories>$(WindowsSDK_WindowsMetadata);$(AdditionalUsingDirectories)</AdditionalUsingDirectories>

